modules/detect_target/detect_target_brightspot.py

```diff
@@ -0,0 +1,110 @@
+"""
+Detects bright spots generated by an IR camera.
+"""
+
+import time
+
+import cv2
+import numpy as np
+
+from . import base_detect_target
+from .. import image_and_time
+from .. import detections_and_time
+
+
+BRIGHTSPOT_THRESHOLD = 240
+
+
+class DetectTargetBrightspot(base_detect_target.BaseDetectTarget):
+    """
+    Detects bright spots in images.
+    """
+
+    def __init__(
```

Comment:
Please copy the more updated function signature from detect_target_factory.py. You should have everything that's in that create function. Also see detect_target_ultralytics.py to see example. (Basically you're gonna want to have a logger)